fanout/propose_fanout.js

```diff
@@ -0,0 +1,248 @@
+const asyncAuto = require('async/auto');
+const asyncEach = require('async/each');
+const asyncMap = require('async/map');
+
+const {connectPeer} = require('ln-sync');
+const {createChainAddress} = require('ln-service');
+const {fundPsbt} = require('ln-service');
+const {decodePsbt} = require('psbt');
+const {getUtxos} = require('ln-service');
+const {lockUtxo} = require('ln-service');
+const {returnResult} = require('asyncjs-util');
+const {unlockUtxo} = require('ln-service');
+const tinysecp = require('tiny-secp256k1');
+
+const asOutpoint = n => `${n.transaction_id}:${n.transaction_vout}`;
+const {isArray} = Array;
+const isEven = n => !(n % 2);
+const isNumber = n => !isNaN(n);
+const nestedSegWitAddressFormat = 'np2wpkh';
+const nestedSegWitPath = "m/49'/";
+
+/** Fund and propose a fanout to a peer
```

Comment:
This isn't really named correctly, you aren't proposing a fanout to a peer you are just creating fanout funding output scripts here